IS7024Project/Pages/Index.cshtml

```diff
@@ -98,34 +99,16 @@
                 <tr>
                     <td>@crime.IncidentNo</td>
                     <td>@crime.Offense</td>
-                    <td>@crime.Location</td>
+                    <td>@Regex.Replace(@Regex.Replace(@crime.Location.ToString(), "[\\d-]", string.Empty), "([a-z])([A-Z])", "$1 $2")</td>
                     <td>@crime.DateFrom</td>
                     <td>@crime.Dayofweek</td>
-                    <td>@crime.CommunityCouncilNeighborhood</td>
-                    <td>@crime.CpdNeighborhood</td>
+                    <td>@Regex.Replace(@crime.CommunityCouncilNeighborhood.ToString(), "([a-z])([A-Z])", "$1 $2")</td>
+                    <td>@Regex.Replace(@crime.CpdNeighborhood.ToString(), "([a-z])([A-Z])", "$1 $2")</td>
```

Comment:
Again, for these two lines I used Regex to convert the methods to strings and then have the code check for a lowercase letter before a uppercase one and if that's the case then add a space.